reference/api/tasks.md

```diff
@@ -49,99 +63,91 @@ List all tasks globally, regardless of index. The `task` objects are contained i
+For each call to this endpoint, the response will include the `next` field: this value should be passed to `from` to view the next "page" of results. When the value of `next` is `null`, there are no more tasks to view.
 
-List all [tasks](/learn/advanced/asynchronous_operations.md) for a given [index](/learn/core_concepts/indexes.md). The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+This command returns tasks two at a time starting from task `uid` `10`.
 
-### Example
+```bash
+curl -X GET 'http://localhost:7700/tasks?limit=2&from=10
+```
 
-<CodeSamples id="get_all_tasks_by_index_1" />
-
-#### Response: `200 Ok`
+**Response:**
 
 ```json
 {
-    "results": [
-        {
-            "uid": 1,
-            "indexUid": "movies",
-            "status": "enqueued",
-            "type": "documentAddition",
-            "duration": null,
-            "enqueuedAt": "2021-08-12T10:00:00.000000Z",
-            "startedAt": null,
-            "finishedAt": null
-        },
-        {
-            "uid": 0,
-            "indexUid": "movies",
-            "status": "succeeded",
-            "type": "documentAddition",
-            "details": { 
-                    "receivedDocuments": 100,
-                    "indexedDocuments": 100
-            },
-            "duration": "PT16S",
-            "enqueuedAt": "2021-08-11T09:25:53.000000Z",
-            "startedAt": "2021-08-11T10:03:00.000000Z",
-            "finishedAt": "2021-08-11T10:03:16.000000Z"
-        }
-    ]
+  "results": [
+    {
+      "uid": 10,
+      "indexUid": "elements",
+      "status": "succeeded",
+      "type": "indexCreation",
+      "details": {
+        "primaryKey": null
+      },
+      "duration": "PT0.006034S",
+      "enqueuedAt": "2022-06-20T13:41:42.446908Z",
+      "startedAt": "2022-06-20T13:41:42.447477Z",
+      "finishedAt": "2022-06-20T13:41:42.453511Z"
+    },
+    {
+      "uid": 9,
+      "indexUid": "particles",
+      "status": "succeeded",
+      "type": "indexCreation",
+      "details": {
+        "primaryKey": null
+      },
+      "duration": "PT0.007317S",
+      "enqueuedAt": "2022-06-20T13:41:31.841575Z",
+      "startedAt": "2022-06-20T13:41:31.842116Z",
+      "finishedAt": "2022-06-20T13:41:31.849433Z"
+    }
+  ],
+  "limit": 2,
+  "from": 10,
+  "next": 8
 }
 ```
 
-## Get task by index
+To view the next page of results, you would repeat the same query, replacing the value of `from` with the value of `next`:
 
-<RouteHighlighter method="GET" route="/indexes/{index_uid}/tasks/{task_uid}"/>
+```bash
+curl -X GET 'http://localhost:7700/tasks?limit=2&from=8
+```
 
-Get a single [task](/learn/advanced/asynchronous_operations.md) in a given [index](/learn/core_concepts/indexes.md). Both the index [`uid`](/learn/core_concepts/indexes.md#index-uid) and task [`uid`](/learn/advanced/asynchronous_operations.md#response) are required.
+When the returned value of `next` is `null`, you have reached the final page of results.
```

Comment:
Personally, I don't think it's necessary: the pagination will be perfectly logical. If a new task is added, it receives the highest sequential `uid`, putting it last in the pagination. It will not affect the order or positioning of subsequent pages, no matter when it occurs.